src/Hyperion.Benchmarks/Program.cs

```diff
@@ -8,8 +8,16 @@ class Program
     {
         static void Main(string[] args)
         {
-            var benchmark = BenchmarkSwitcher.FromAssembly(Assembly.GetExecutingAssembly());
-            benchmark.RunAll();
+            var benchmark = BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly);
+
+            if (args.Length == 0)
+            {
+                benchmark.RunAll();
+            }
+            else
+            {
+                benchmark.Run(args);
```

Comment:
Slight changes so we can pass arguments to BenchmarkDotNet from command line.
`RunAll()` is just a shortcut to call `Run()` with pre-populated filter argument.